server/test/unit/server/webservices/views/test_util.py

```diff
@@ -54,3 +54,14 @@ def test_generate_json_response_with_pulp_encoder(self, mock_json):
         test_content = {'foo': 'bar'}
         util.generate_json_response_with_pulp_encoder(test_content)
         mock_json.dumps.assert_called_once_with(test_content, default=pulp_json_encoder)
+
+    def test_generate_response_redirect(self):
+        """
+        Test HttpResponseRedirect.
+        """
+        test_content = {'foo': 'bar'}
+        response = util.generate_json_response_with_pulp_encoder(test_content)
```

Comment:
Can this call to `utils.generate_json_response_with_pulp_encoder` build the object manually? A simple HttpResponse(content=test_content) should suffice. This way the unit test won't be affected by breakages in `utils.generate_json_response_with_pulp_encoder`.